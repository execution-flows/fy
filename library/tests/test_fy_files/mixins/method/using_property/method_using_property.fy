method greet using greeting:
    with property greeting

    def (argument: str) -> None:
        print(self._greeting)
