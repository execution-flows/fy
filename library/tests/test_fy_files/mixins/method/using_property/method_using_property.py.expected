import abc

from test_fy_files.mixins.method.using_property.abc import With_Greeting_PropertyMixin_ABC

import datetime


class Greet_UsingGreeting_MethodMixin(
    # Property_mixins
    With_Greeting_PropertyMixin_ABC,
    abc.ABC,
):
    def _greet(self, argument: str) -> None:
        datetime.datetime.now()
        print(self._greeting)
